brush/src/components/Map/TilemapLayer/index.js

```diff
@@ -18,56 +18,57 @@ const listCoordinates = (height, width) => {
   return coordinates
 }
 
-const TilemapLayer = ({ setSelectedPointInfos, vision }) => {
-  const {
-    infos: {
-      index,
-      tilemap: {
-        height,
-        scheme,
-        width,
+// This component needs to be a class because we need to get its ref
```

Comment:
You tried to use a [`useRef` hook](https://reactjs.org/docs/hooks-reference.html#useref) ?

Reply:
Yeah. I could use `useRef hook`, but I still need to use a class:
```
react-pixi.module-dev.js:2531 Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?
```